src/cascadia/UnitTests_TerminalCore/SelectionTest.cpp

```diff
@@ -513,12 +513,12 @@ namespace TerminalCoreUnitTests
             // Simulate move to (x,y) = (5,10)
             //
             // buffer: doubleClickMe dragThroughHere
-            //         ^                ^
-            //       finish            start
+            //          ^               ^
+            //        finish           start

```

Comment:
wat?

what was this testing, and does moving to the `o` instead of the `d` maintain that invariant?

Reply:
`SetSelectionEnd` below was already targeting (5,10), so this just fixed the comment above to be correct (same with line 505 above).